buy-resell/products/3rd-party/FinanzOnline Management.md

```diff
@@ -0,0 +1,101 @@
+|:----------------------:|-------------------------------------------------------------------------------------------------------------------------------------|
+|![Number 1](../../../images/numbers/circle-1o.png) |Enter the fiskaltrust.Portal and select `PosDealer` / `FinanzOnline Management`.  |
+|![Number 2](../../../images/Numbers/circle-2o.png) |For further investigations on a PosOperator you can use the link to switch to a certain PosOperators account. Select `Tools` / `FinanzOnline notifications` to check the state of forwarding or the completeness of the registration data for FinanzOnline.  |
+|![Number 3](../../../images/Numbers/circle-3o.png) |Check the actual status of the FinanzOnline Connection at `State of FinanzOnline Notifications`.  |
+
+#### Send notifications for a single Queue or SCU
+
+![](../../../buy-resell/images/FON-Management-2.png)
+
+| steps | description                                                                                                                |
+|:----------------------:|-------------------------------------------------------------------------------------------------------------------------------------|
+|![Number 1](../../../images/Numbers/circle-1o.png) |Select the collapser for the details of one of your PosOperators. |
+|![Number 2](../../../images/Numbers/circle-2o.png) |Open the desired tab for details. You can check the amount of available entitlements. </br>If none are available, the tab shows a message like `No entitlements to display`.  |
+|![Number 3](../../../images/Numbers/circle-3o.png) |If necessary, reduce the number of search results displayed by filter criteria like </br>`Cash register identification number`. |
+|![Number 4](../../../images/Numbers/circle-4o.png) |For a single `Queue` or on the corresponding tab for a single `SCU` you can </br>`Start a status check at FinanzOnline`. |
+
+
+#### Send notifications for a single PosOperator
+
+![](../../../buy-resell/images/FON-Management-3.png)
+
+| steps | description                                                                                                                |
+|:----------------------:|-------------------------------------------------------------------------------------------------------------------------------------|
+|![Number 1](../../../images/Numbers/circle-1o.png) |Select `Execute FinanzOnline status update` for of one of your PosOperators. |
+|![Number 2](../../../images/Numbers/circle-2o.png) |You have to confirm to send all notifications for queues and SCUs for the selected PosOperator.  |
+
+#### Send notifications for all PosOperators
+
+![](../../../buy-resell/images/FON-Management-4.png)
+
+ With the action `Execute FinanzOnline status update for all PosOperators` all messages to queues and/or SCUs for all PosOperators will be sent to FinanzOnline again and claims to messages will be consumed as well. We recommend to check the PosOperators status individually a first. 
```

Comment:
Status update is only a status request to FOn, we basically check the status of the Queue/SCU in the FOn Database (e.g. if any changes have been done manually in FOn). We don't really send messages